rules/renaming/tests/Rector/MethodCall/RenameMethodRector/config/configured_rule.php

```diff
@@ -25,6 +25,11 @@
                     'old',
                     'new'
                 ),
+                new MethodCallRename(
+                    'Rector\Renaming\Tests\Rector\MethodCall\RenameMethodRector\Fixture\*WildcardSubscriber',
```

Comment:
Could you provide `https://3v4l.org` proof that the `fnmatch()` work with expected class?
That would narrow the scope for the issue

Reply:
Good one. I checked, and it does work: https://3v4l.org/4tCj4

Reply:
The problem is that `getOldClass()` still contains the wildcard and is then passed to `ClassManipulator::hasParentMethodOrInterface`.

<img width="1019" alt="Screenshot 2021-02-16 at 09 13 46@2x" src="https://user-images.githubusercontent.com/104180/108035524-5797a400-7037-11eb-8a8c-8b84e6c8dbd3.png">

Reply:
What is needed to fix it?

Reply:
I don't know. 

@samsonasik added this: https://github.com/rectorphp/rector/blob/8d58a6dd87a0618b70b3cc6be979f4f6f9d843b1/rules/renaming/src/Rector/MethodCall/RenameMethodRector.php#L88-L94

But that doesn't have support for `fnmatch`. 

I think that we should not use `$methodCallRename->getOldClass()` as it can be a FQCN or wildcard matching.

Instead, we should check if the Class `$node` has interfaces that implement `$methodCallRename->getOldMethod()` method. If that's the case, it should not rename. But I tried this already and I couldn't get it to work.